sky/execution.py

```diff
@@ -253,7 +256,7 @@ def launch(
     retry_until_up: bool = False,
     idle_minutes_to_autostop: Optional[int] = None,
     dryrun: bool = False,
-    teardown: bool = False,
+    terminate: bool = False,
```

Comment:
Just a note - no action needed:

May break existing users, but that's ok pre 1.0.

If we want to be user-friendlier we can try to catch whether `teardown` is passed and hint to rename the arg. However this makes the code harder to read/maintain. I think the current way is ok.

Reply:
Good point! I am leaning toward not including the code for the hint for now, but we should definitely keep these changes in mind, when we have more users.